content/Ruby doesn't scale - Anthony Figueroa.md

```diff
@@ -0,0 +1,74 @@
+
+In this article, I focus on Ruby, but the information is valid for almost any programming language. If you typically benchmark Ruby against other languages like Python or C++, it's probably slower in most contexts. 
+
+The *real* question isn't how long it takes or how many resources it consumes to run some algorithms like regex redux, binary tree searches, or reading DNA sequences.
+
+## Scalability questions
+
+* How soon you want to go to the market?
+* How can you break down your platform into smaller more cohesive pieces?
+* Do you have performance requirements? If yes, what's their ultimate goal? A better UX?
+* How do you track your success?
+* Are you building a proof of concept? A mission-critical application?
+* Is scalability important right from day 0?
+* If you succeed, how will your platform evolve? Is user growth an objective?
+
+## Every project is different
+
+The answers to these questions might seem obvious, but they're not. Each of the following project scenarios presents entirely different answers to the scalability questions:
+
+* Create a disposable MVP that needs to go to the market in eight weeks to test a business hypothesis.
+* Build an app to support the processes and logic of a well-established business with over 100 million in revenue.
+* Create a scalable, high-quality MVP that will be released by a VC-funded startup with a high marketing budget.
+* Re-engineer and release a version 2 of an existing platform that's having scalability problems due to high growth of its user base.
+
+To know where you stand, you need to ask the scalability questions in all of these scenarios. In each case, you'll get radically different answers.
+
+## Working with Ruby
+
+I've worked on hundreds of applications in my career. Probably most of the startup work I've done has Ruby on Rails components. And do you know what? It's a fantastic framework for going fast to market with high-quality maintainable code, thanks to the company's strong conventions and standards. Ruby has great tools to change, adapt, and pivot your idea. It's fantastic for most startups who are willing to test their minimum viable product (MVP). In the last few years, we've taken the approach of building APIs and lightweight clients, mostly by using ReactJS.
+
+## Increase your chance of winning
+
+Let's be honest. Most startups fail. But if you go to market fast, you can learn and iterate more quickly. Hopefully, you find a product-market fit before you run out of money. Our job is to make sure a startup doesn't fail because of poor technical execution. We can say that we successfully built a lot of failed startups. But we also built some wildly successful ones.
+
+## Platform scaling
+
+In the best case, a platform might become a sustainable business or move in the direction of an exit because of user growth. At this point, we probably need to scale the platform, which is the best problem to have.
+We usually face one of the following two possible scenarios.
```

Comment:
This sentence sounds weird to me, how about: 
`We usually face one of the following scenarios:`
or
`We usually face one of these two scenarios:`